index.html

```diff
@@ -47,7 +47,6 @@
               <input type="password" placeholder="Password" class="form-control">
             </div>
             <button type="submit" class="btn btn-success">Sign in</button>
-              <button type="create" class="btn btn-create">Create Account</button>
```

Comment:
Is this for us to remove the ability to create a new account, or to simulate already being signed in?